pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -648,6 +648,15 @@ func (nc *Controller) doNoExecuteTaintingPass(ctx context.Context) {
 				logger.V(4).Info("Node was in a taint queue, but it's ready now. Ignoring taint request", "node", klog.KRef("", value.Value))
 				return true, 0
 			}
+			pods, err := nc.getPodsAssignedToNode(node.Name)
+			if err != nil {
+				logger.Info("unable to list pods of node", "node", node.Name, "err", err)
+				return false, 50 * time.Millisecond
```

Comment:
We have used `50 * time.Millisecond` many times, can we define it as a constant?